### DISCUSSION TITLE
Is there a way to override all base defaultProps at once?

### DISCUSSION THREAD

**Discussion by GollyJer** [GeekyAnts/NativeBase#5002] [Q&A] [answered]
Posted on StackOverflow. If I get an answer here I'll update there and vice-versa. 
https://stackoverflow.com/q/72134010/25197

Thanks!

**Comment by sanna231** [comment, marked as answer]
Hey @GollyJer , As of now we don't have any hook that can update all default props at once. Thanks for your suggestions. We'll have a discussion around it.
